scanpy/tests/test_package_structure.py

```diff
@@ -41,5 +57,11 @@ def test_function_headers(f):
         raise SyntaxError(msg, (filename, lineno, 2, text))
 
 
-def test_plot_doc_signatures():
-    pass
+def test_metadata(tmp_path, in_project_dir):
+    import flit_core.buildapi
+
+    flit_core.buildapi.prepare_metadata_for_build_wheel(tmp_path)
+
+    metadata_path = next(tmp_path.glob('*.dist-info')) / 'METADATA'
+    metadata = email.message_from_bytes(metadata_path.read_bytes())
+    assert not metadata.defects
```

Comment:
Is this covered by the build and `twine check` task?

Reply:
Not necessarily, `twine check` just

> Checks whether your distribution’s long description will render correctly on PyPI.

so might be that it misses other problems.